x/twap/store.go

```diff
@@ -117,6 +121,19 @@ func (k Keeper) pruneRecordsBeforeTimeButNewest(ctx sdk.Context, lastKeptTime ti
 		store.Delete(timeIndexKey)
 		poolIndexKey := types.FormatHistoricalPoolIndexTWAPKeyFromStrTime(poolId, asset0, asset1, timeS)
 		store.Delete(poolIndexKey)
+
+		numPruned++
```

Comment:
Let's make this += 2 since we are pruning two records per loop

Reply:
https://github.com/osmosis-labs/osmosis/pull/7427/commits/a078f0bf454ca54cc986f7a28eb9d3f249cd112b